examples/redis/client.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const opentelemetry = require('@opentelemetry/core');
+const types = require('@opentelemetry/types');
+const config = require('./setup');
+config.setupTracerAndExporters('redis-client-service');
+const tracer = opentelemetry.getTracer();
+const axios = require('axios').default;
+
+function makeRequest() {
+    const span = tracer.startSpan('client.makeRequest()', {
+        parent: tracer.getCurrentSpan(),
+        kind: types.SpanKind.CLIENT
+    });
+
+    tracer.withSpan(span, async () => {
+        try {
+            const res = await axios.get('http://localhost:8080/run_test');
```

Comment:
I am currently using axios since it removes a lot of the unimportant "noise code" in the sample. Currently nothing interesting happens in `client.js` other than kicking off the trace, so I would prefer to keep axios. WDYT?

Reply:
My thought was just to have consistency within all the examples and codebase. So far we don't use axios, but rather just standard libraries as then the user can choose what he wants. If that's the way to go then fine I just think we should have consistency :).

Reply:
I actually don't mind the use of axios here. In fact, it shows that third party libraries can be used. These are meant to be examples of real world use and almost nobody uses the `http` library directly. These aren't shipped to customers in the bundle so additional dependencies don't have as big a burden as they do in the packages